android/source/VideoLocker/res/anim/no_transition.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android" >
+
+    <translate
+        android:duration="500"
```

Comment:
Basically, for sign-in and register screens, we have appear-from-bottom animation. But, while next screen (sign-in or register) is appearing, current screen should be still (no-animation). 
Appear animation is of 500 ms duration, so no-transition should also be of same duration. Otherwise, we might see blank screen in the background while next screen (sign-in or register) is appearing from bottom.